Node/samples/LuisActionBinding/bot/app.js

```diff
@@ -0,0 +1,75 @@
+require('dotenv-extended').load({ path: '../.env' });
+
+var builder = require('botbuilder');
+var restify = require('restify');
+
+var LuisActions = require('botbuilder-cognitiveservices').LuisActionBinding;
+var SampleActions = require('../all');
+var LuisModelUrl = process.env.LUIS_MODEL_URL;
+
+var server = restify.createServer();
+server.listen(process.env.port || process.env.PORT || 3978, function () {
+    console.log('%s listening to %s', server.name, server.url);
+});
+
+var connector = new builder.ChatConnector({
+    appId: process.env.MICROSOFT_APP_ID,
+    appPassword: process.env.MICROSOFT_APP_PASSWORD
+});
+server.post('/api/messages', connector.listen());
+
+var bot = new builder.UniversalBot(connector);
```

Comment:
Can you modify this sample (and the other bot samples) so that they use in memory storage instead of using the Bot State API? Some examples can be found in [PR 3792](https://github.com/Microsoft/BotBuilder/pull/3792/files) in the main BotBuilder repo. Thanks!